src/lib/coda_base/transaction_union_payload.ml

```diff
@@ -59,7 +59,9 @@ module Body = struct
         | Payment ->
             (Amount.zero, max_amount_without_overflow)
         | Stake_delegation ->
-            (Amount.zero, max_amount_without_overflow)
+            (Amount.zero, Amount.zero)
```

Comment:
why did the result for `Stake_delegation` change?

Reply:
There's no amount associated with `Stake_delegation`, and giving Quickcheck a 1-element range should stop it from wasting any time exercising different `amount` values that will have no impact.